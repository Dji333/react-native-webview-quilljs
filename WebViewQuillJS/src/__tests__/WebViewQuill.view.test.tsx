import WebViewQuillView from "../WebViewQuill.view";
import { render } from "@testing-library/react-native";
import * as React from "react";

// End up getting a "ReferenceError: React is not defined" error if I don't declare React globally
// @ts-ignore
global.React = React;

const props={
  debugMessages:[], 
  handleMessage:()=>{}
}

beforeEach(() => {});

describe("WebViewQuill.view.test.tsx", () => {
  it("should render", () => {
    const { container  } = render(<WebViewQuillView {...props} />);
    expect(container).toMatchSnapshot()
  });
});
