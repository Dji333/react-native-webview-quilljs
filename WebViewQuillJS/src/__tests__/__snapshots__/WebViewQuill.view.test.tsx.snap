// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WebViewQuill.view.test.tsx should render 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
/>
`;
